src/components/LoginPanel.tsx

```diff
@@ -40,6 +40,10 @@ const MiddleDiv = styled.div`
         -webkit-text-fill-color: ${(props) => props.theme.main.foreground[2]} !important;
     }
 
+    & button:hover {
+        color: ${(props) => props.theme.main.foreground[2]};
+    }
+
```

Comment:
Changing the media queries below should sort the padding issue (think especially the 525px one)